README.md

```diff
@@ -1,31 +1,2 @@
 # rcl_interfaces
-This package contains the messages and services which ROS client libraries will use under the hood to
-communicate higher level concepts such as parameters.
-
-
-## Parameter Groups
-Parameters are contained in groups.
-The default group is '/'.
-It behaves like a filepath, where you can nest sub-groups within groups.
-
-## Standard topics for parameters
-
-The ROS API for a node will be as follows inside the node's namespace.
-
-### Topics:
- * `parameter_events`: `ParameterEvent`
-  * This topic provides a way to subscribe to all parameter updates occuring on the node, including addition removal and changes in value. Every atomic change will be published separately.
- * `parameter_event_descriptors`: `ParameterEventDescriptors`
-  * This topic provides a way to subscribe to all parameter updates occuring on the node, including addition removal and changes in value.
-    Every atomic change will be published separately. This is provided if large parameter values are expected to slow down the system.
-
-### Services:
-
- * `get_parameters`: `GetParameters`
-  * The service to get the value of parameters which are set on this node.
- * `has_parameters`: `HasParameters`
-  * Query this node if specific parameters are set.
- * `list_parameters`: `ListParameters`
-  * List the parameters on this node matching the filters.
- * `set_parameters`: `SetParameters`
-  * Set parameters on this node.
+A set of packages which contain interface files (.msg and .srv) used for the internal implementation and testing of the client libraries.
```

Comment:
I wouldn't consider those "internal". Most of them define public interfaces, no?

Reply:
"used for the implementation and testing" then ?

Reply:
"defining the interface across client libraries"?

Reply:
Hmm not sure because as you said it is user facing and currently most of them are not used to communicate across client libraries given that they're used mostly/(only?) in rclcpp.

@ros2/team maybe some native speakers have some insights ?

Reply:
They're definitely public and I would say their primary use is implementation of rcl concepts rather than testing, though they could be used for that too I guess.

"This repository contains a set of packages which themselves contain interface files (i.e. `.msg` and `.srv` files) used to implement client library concepts like parameters."

Reply:
my goal here is to give a description of the content of the repository and not only describe the `rcl_interfaces` package, that's why I talked about testing (given that this repo also contains test_msgs).

I'm fine with "This repository contains a set of packages which themselves contain interface files (i.e. .msg and .srv files) used to implement client library concepts" I just wanted to clarify the scope

Reply:
Oh, I was confused by the diff and I didn't know there were test messages here. In that case I agree, something like:

"This repository contains a set of packages which themselves contain interface files (i.e. `.msg` and `.srv` files) which are used both to implement client library concepts and in testing."

Or something like that.

Reply:
I've updated it with approximately the verbiage of above.